### ISSUE TITLE
[Bug report] Plugin doesn't support AGP 8

### ISSUE THREAD

**Issue by TatsuUkraine** [fluttercandies/flutter_image_editor#120]
### Version

1.3.0

### Platforms

Android

### Device Model

All

### flutter info

```sh
[✓] Flutter (Channel stable, 3.16.6, on macOS 13.5.2 22G91 darwin-arm64, locale ru)
    • Flutter version 3.16.6 on channel stable at ...
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 46787ee49c (2 days ago), 2024-01-09 14:36:07 -0800
    • Engine revision 3f3e560236
    • Dart version 3.2.3
    • DevTools version 2.28.4

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
    • Android SDK at ...
    • Platform android-34, build-tools 33.0.0
    • ANDROID_HOME = ...
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 17.0.7+0-17.0.7b1000.6-10550314)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 15.2)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 15C500b
    • CocoaPods version 1.13.0

[✓] Android Studio (version 2023.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.7+0-17.0.7b1000.6-10550314)

[✓] Network resources
    • All expected network resources are available.
```

### How to reproduce?

Configure project to use Gradle 8

### Logs

```sh
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':image_editor_common'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.

     If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.
```

**Comment by martinogarrizzo5** [comment]
+1

**Comment by waveproject0** [comment]
I am facing the same issue. It would be really helpful if this plugin can support AGP 8.

**Comment by wemgl** [comment]
Any news on supporting AGP 8?

**Comment by CaiJingLong** [comment]
Merged in #117.

**Comment by scolnet** [comment]
Hello,

@CaiJingLong, it seems that the new release 1.4.0 use a version of image_editor_common (1.0.1) which do not use the fix to support AGP 8
